system-int-test/test/cmr/system_int_test/ingest/service_ingest_test.clj

```diff
@@ -119,15 +120,23 @@
 ;; Service with concept-id ingest and update scenarios.
 (deftest service-w-concept-id-ingest-test
   (let [supplied-concept-id "S1000-PROV1"
-        concept (service-util/make-service-concept
-                 {:concept-id supplied-concept-id
-                  :native-id "Atlantic-1"})]
+        metadata {:concept-id supplied-concept-id
+                  :native-id "Atlantic-1"}
+        concept (service-util/make-service-concept metadata)]
     (testing "ingest of a new service concept with concept-id present"
       (let [{:keys [concept-id revision-id]} (ingest/ingest-concept concept)]
         (is (mdb/concept-exists-in-mdb? concept-id revision-id))
         (is (= [supplied-concept-id 1] [concept-id revision-id]))))
-
-    (testing "Update the concept with the concept-id"
+    (testing "ingest of same native id and different providers is allowed"
+      (let [concept2-id "S1000-PROV2"
+            concept2 (service-util/make-service-concept
+                      (assoc metadata :provider-id "PROV2"
+                                      :concept-id concept2-id))
+            {:keys [concept2-id revision2-id]} (ingest/ingest-concept concept2)]
```

Comment:
Change this line to `{:keys [concept-id revision-id]} (ingest/ingest-concept concept2)]`

Reply:
Fixed -- thanks!